### PR TITLE
Docs: Rename Operator and user guide to Manage

### LINKED ISSUE #5225: Docs: Change "Operator and user guide" to "Manage"

### PR DESCRIPTION
**Pull request by osg-grafana**
Fix #5225; this work aligns with #4274.

For a visual of the cohesion that this brings across products, see Tempo’s v2.1.x TOC, which contains a top-level `Manage` menuTitle:
image

### CODE REVIEW

**Comment by osg-grafana** [comment]
@jdbaldry, here you go. Please let me know if you want me to do anything else before you test the newest version of `mv-manager`.

**Comment by osg-grafana** [comment]
> @jdbaldry, here you go. Please let me know if you want me to do anything else before you test the newest version of `mv-manager`.

For example, does `mv-manager` handle renames of images? Can we apply aliases to images somehow?

**Comment by jdbaldry** [comment]
> > @jdbaldry, here you go. Please let me know if you want me to do anything else before you test the newest version of `mv-manager`.
> 
> For example, does `mv-manager` handle renames of images? Can we apply aliases to images somehow?

`mv-manager` does not currently support image renaming but I believe this shouldn't be a problem if you are following the guidelines in https://grafana.com/docs/writers-toolkit/writing-guide/image-guidelines/ or using images only stored in the bundle.

**Comment by osg-grafana** [comment]
> I think these conflicts should be resolved before we start rewriting links. Are you able to do that?

I agree. Let me see what happens if I pull `main` into my branch.

**Comment by krajorama** [comment]
I've ran
```
git grep operators-guide | grep -v 'CHANGELOG.md' | grep -v -E '^docs/sources/mimir/.*:  -' | less
```

and it gives me 270 hits for mentions of operators-guide
105 from the helm chart docs
144 from the compiled mixins (alerts refer to the doc)
21 others, scripts, readme:
```
README.md:For information about how to deploy Grafana Mimir, refer to [Deploy Grafana Mimir](https://grafana.com/docs/mimir/latest/operators-guide/deploy-grafana-mimir/).
README.md:1. [An overview of Grafana Mimir’s architecture](https://grafana.com/docs/mimir/latest/operators-guide/architecture/)
README.md:1. [Configure Grafana Mimir](https://grafana.com/docs/mimir/latest/operators-guide/configure/)
README.md:1. [Run Grafana Mimir in production](https://grafana.com/docs/mimir/latest/operators-guide/run-production-environment/)
development/README.md:[deployment-modes]: https://grafana.com/docs/mimir/latest/operators-guide/architecture/deployment-modes/
docs/internal/contributing/README.md:- Update the runbook in `docs/sources/mimir/operators-guide/mimir-runbooks/_index.md` with details about why the error happens, and if possible how to address it.
docs/sources/mimir/references/configuration-parameters/index.template:- ../operators-guide/configure/reference-configuration-parameters/
docs/sources/mimir/references/configuration-parameters/index.template:- ../operators-guide/configuring/reference-configuration-parameters/
operations/mimir-mixin-tools/screenshots/run.sh:  -v "${SCRIPT_DIR}/../../../docs/sources/mimir/operators-guide/monitor-grafana-mimir/dashboards:/output" \
operations/mimir-mixin/alerts/alertmanager.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
operations/mimir-mixin/alerts/alerts.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
operations/mimir-mixin/alerts/autoscaling.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
operations/mimir-mixin/alerts/blocks.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
operations/mimir-mixin/alerts/compactor.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
operations/mimir-mixin/alerts/continuous-test.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
operations/mimir-mixin/config.libsonnet:    // docs/sources/mimir/operators-guide/monitoring-grafana-mimir/requirements.md
operations/mimir/README.md:The documentation for the Mimir Jsonnet is published at [https://grafana.com/docs/mimir/next/operators-guide/deploy-grafana-mimir/jsonnet/](https://grafana.com/docs/mimir/next/operators-guide/deploy-grafana-mimir/jsonnet/).
packaging/nfpm/mimir/runtime_config.yml:# https://grafana.com/docs/mimir/latest/operators-guide/configure/about-runtime-configuration/
pkg/mimirtool/config/descriptors/gem-v2.6.0.json:              "desc": "Comma separated addresses list in DNS Service Discovery format: https://grafana.com/docs/mimir/latest/operators-guide/configuring/about-dns-service-discovery/",
pkg/mimirtool/config/descriptors/gem-v2.6.0.json:                          "desc": "Comma separated addresses list in DNS Service Discovery format: https://grafana.com/docs/mimir/latest/operators-guide/configuring/about-dns-service-discovery/",
tools/lint-runbooks.sh:  if ! grep -q "# ${ID}$" "${SCRIPT_DIR}/../docs/sources/mimir/operators-guide/mimir-runbooks/_index.md"; then
```

I feel like we could fix these as well to get a consistent state on main after the PR. WDYT @osg-grafana

**Comment by osg-grafana** [comment]
> I've ran
> 
> ```
> git grep operators-guide | grep -v 'CHANGELOG.md' | grep -v -E '^docs/sources/mimir/.*:  -' | less
> ```
> 
> and it gives me 270 hits for mentions of operators-guide 105 from the helm chart docs 144 from the compiled mixins (alerts refer to the doc) 21 others, scripts, readme:
> 
> ```
> README.md:For information about how to deploy Grafana Mimir, refer to [Deploy Grafana Mimir](https://grafana.com/docs/mimir/latest/operators-guide/deploy-grafana-mimir/).
> README.md:1. [An overview of Grafana Mimir’s architecture](https://grafana.com/docs/mimir/latest/operators-guide/architecture/)
> README.md:1. [Configure Grafana Mimir](https://grafana.com/docs/mimir/latest/operators-guide/configure/)
> README.md:1. [Run Grafana Mimir in production](https://grafana.com/docs/mimir/latest/operators-guide/run-production-environment/)
> development/README.md:[deployment-modes]: https://grafana.com/docs/mimir/latest/operators-guide/architecture/deployment-modes/
> docs/internal/contributing/README.md:- Update the runbook in `docs/sources/mimir/operators-guide/mimir-runbooks/_index.md` with details about why the error happens, and if possible how to address it.
> docs/sources/mimir/references/configuration-parameters/index.template:- ../operators-guide/configure/reference-configuration-parameters/
> docs/sources/mimir/references/configuration-parameters/index.template:- ../operators-guide/configuring/reference-configuration-parameters/
> operations/mimir-mixin-tools/screenshots/run.sh:  -v "${SCRIPT_DIR}/../../../docs/sources/mimir/operators-guide/monitor-grafana-mimir/dashboards:/output" \
> operations/mimir-mixin/alerts/alertmanager.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
> operations/mimir-mixin/alerts/alerts.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
> operations/mimir-mixin/alerts/autoscaling.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
> operations/mimir-mixin/alerts/blocks.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
> operations/mimir-mixin/alerts/compactor.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
> operations/mimir-mixin/alerts/continuous-test.libsonnet:  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', alertGroups),
> operations/mimir-mixin/config.libsonnet:    // docs/sources/mimir/operators-guide/monitoring-grafana-mimir/requirements.md
> operations/mimir/README.md:The documentation for the Mimir Jsonnet is published at [https://grafana.com/docs/mimir/next/operators-guide/deploy-grafana-mimir/jsonnet/](https://grafana.com/docs/mimir/next/operators-guide/deploy-grafana-mimir/jsonnet/).
> packaging/nfpm/mimir/runtime_config.yml:# https://grafana.com/docs/mimir/latest/operators-guide/configure/about-runtime-configuration/
> pkg/mimirtool/config/descriptors/gem-v2.6.0.json:              "desc": "Comma separated addresses list in DNS Service Discovery format: https://grafana.com/docs/mimir/latest/operators-guide/configuring/about-dns-service-discovery/",
> pkg/mimirtool/config/descriptors/gem-v2.6.0.json:                          "desc": "Comma separated addresses list in DNS Service Discovery format: https://grafana.com/docs/mimir/latest/operators-guide/configuring/about-dns-service-discovery/",
> tools/lint-runbooks.sh:  if ! grep -q "# ${ID}$" "${SCRIPT_DIR}/../docs/sources/mimir/operators-guide/mimir-runbooks/_index.md"; then
> ```
> 
> I feel like we could fix these as well to get a consistent state on main after the PR. WDYT @osg-grafana

I was wondering about past `CHANGELOG.md` files, which on a good night’s sleep do not matter because they will remain as-is. I think we should change all the current (`main`) links to align with the new location of the files. Thank you for grepping! :) And @brendamuir brought up Alerting docs pointing to Mimir docs yesterday. We can cover this same concern before merging into `main`.

File: operations/mimir/README.md
Comment (krajorama) [inline operations/mimir/README.md:4]: Discussed IRL, this should be more generic, point to top of docs so we don't have to maintain.

Comment (krajorama) [reply operations/mimir/README.md:4]:
```suggestion
For documentation about Mimir Jsonnet, go to [https://grafana.com/docs/mimir/next/](https://grafana.com/docs/mimir/next/).
```

**Review by krajorama — APPROVED** [review]
LGTM, once published - let's not forget to update the top level  README to reflect the changes

**Pull request merged**
